CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Added
+* `crud.get_storages_state` function to get storages initialization state (#229).
```

Comment:
Nits:

1. AFAIK usage of plural nouns in a function/field name is discouraged.
2. We usually leave getters without `get_` prefix.
3. tarantool uoften uses `.info()` in this sense, so, maybe, `crud.storage_info()`?

Reply:
Done.